fastlane/lib/versions.rb

```diff
@@ -18,6 +18,11 @@ def current_build_number(**args)
   end
 end
 
+# get the current build number from the environment
+def build_number()
+  return ENV['TRAVIS_BUILD_NUMBER'] || ENV['CIRCLE_BUILD_NUM']
```

Comment:
Don't `return`, just put everything that follows after that.  (`return` is not required in Ruby, provided you are at the end of a block or method.)

```ruby
def build_number
  ENV['TRAVIS_BUILD_NUMBER'] || ENV['CIRCLE_BUILD_NUM']
end
```
should be sufficient.